pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -67,10 +67,10 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>1.0.1</Version>
+      <Version>1.0.3</Version>
```

Comment:
See https://github.com/dotnet/core-setup/pull/1523/files#diff-e2c52444a9da4a5eae65d36e5d3e6abaR51 for the description of why this is needed.